### ISSUE TITLE
Add the ability to generate random messages of a given size

### ISSUE THREAD

**Issue by atoulme** [whiteblock/gossipsub-testing#3]
Allow to create random messages of a given size, with the size being given as a parameter to the function.

**Comment by adam-hanna** [comment]
Agreed.

There's a bytes array that will be used to hold this variable sized data in the message protobuf:
https://github.com/agencyenterprise/gossip-host/blob/6c5adf566b389edd47656e75a229e354c06e1e57/internal/host/pb/message.proto#L7

**Comment by araskachoi** [comment]
@adam-hanna Hey adam, so does this create a message with indicated size (in bytes)? I'm not sure how important it is but should we make the whole message be the indicated size or should we have the message data field be the indicated size? might not matter so much right now

**Comment by adam-hanna** [comment]
Right now, it just reads from a json file. So the json file would need to have a bytes array of the desired size.

I could really easily add flag that replaces the bytes array in the json file with one of a desired size.

I think the entire message size should be the metric we target.

**Comment by araskachoi** [comment]
Yes I agree, the entire size of the message should be the size that we indicated.

Awesome, this will be needed when we will run the msg_size test; where we vary the message size in bytes
